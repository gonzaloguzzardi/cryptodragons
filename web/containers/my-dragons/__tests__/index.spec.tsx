import React from 'react'
import renderer from 'react-test-renderer'

import MyDragons from '..'

test('Renders MyDragons desktop', () => {
  const deviceType = 'desktop'
  const component = renderer.create(<MyDragons deviceType={deviceType} />)

  expect(component).toMatchSnapshot()
})

test('Renders MyDragons mobile', () => {
  const deviceType = 'mobile'
  const component = renderer.create(<MyDragons deviceType={deviceType} />)

  expect(component).toMatchSnapshot()
})
