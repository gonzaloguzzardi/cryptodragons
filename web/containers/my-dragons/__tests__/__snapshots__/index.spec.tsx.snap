// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders MyDragons desktop 1`] = `
<div
  className="container"
>
  <main>
    <AppToolbar
      accountsState={
        Object {
          "connectToProvider": [Function],
          "mainchain_account": null,
          "provider_installed": false,
        }
      }
      deviceType="desktop"
      section="my-dragons"
    />
    <p>
      My Dragons
    </p>
  </main>
</div>
`;

exports[`Renders MyDragons mobile 1`] = `
<div
  className="container"
>
  <main>
    <AppToolbar
      accountsState={
        Object {
          "connectToProvider": [Function],
          "mainchain_account": null,
          "provider_installed": false,
        }
      }
      deviceType="mobile"
      section="my-dragons"
    />
    <p>
      My Dragons
    </p>
  </main>
</div>
`;
